torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -313,6 +313,16 @@ class TORCH_API ProcessGroupNCCL : public Backend {
     friend class ProcessGroupNCCL;
   };
 
+  class DebugInfoCallbackStorer {
```

Comment:
nit: maybe a shorter name?
What about DebugInfoWriter

Reply:
Sure.. Let's use the name Writer, originally I am not quite sure if we need to put FallBack in the name.